setup.py

```diff
@@ -37,10 +35,11 @@
           'scipy',
           'pytest',
           'pytest-cov',
-          'opencv-python'
+          'opencv-python',
+          'tensorflow'
```

Comment:
it is not recommended to put tensorflow in the setup.py.  because users may want to use tensorflow-gpu instead

Reply:
ah, ok